### ISSUE TITLE
Strings in quotes are incorrectly validated against most rules

### ISSUE THREAD

**Issue by ryanmerolle** [aristanetworks/j2lint#15]
I will add to the list of examples below:

**"regex with |"**
```bash
{{ "Hello World" | regex_replace('(o|l)') }}
```

**"regex with |" result**
```bash
test.j2:1 The operator | needs to be enclosed by a single space on each side (operator-enclosed-by-spaces)
```

**Comment by manuwelakanade** [comment]
Thanks @ryanmerolle for this issue. I will work on this.

**Comment by ClausHolbechArista** [comment]
Another example is the `V1` rule incorrectly flagging `   {{ a_non_existing_var | mandatory('The number of ACL entries is above defined maximum!') }}`

**Comment by manuwelakanade** [comment]
This is fixed as part of https://github.com/aristanetworks/j2lint/pull/31 hence closing
